Packs/FeedZoom/pack_metadata.json

```diff
@@ -1,8 +1,8 @@
 {
     "name": "Zoom Feed",
     "description": "Use the Zoom Feed to automatically ingest valid indicators such as CIDRs and DomainGlobs from Zoom. The indicators can then be added to firewall allow lists.",
```

Comment:
Please add the videos property as in https://github.com/demisto/content/blob/master/Packs/MalwareInvestigationAndResponse/pack_metadata.json#L5
and remove the embedded URL in the README so we will support the youtube link natively, and not via markdown.